apps/pipelines/nodes/nodes.py

```
@@ -144,11 +146,29 @@ class LLMResponseMixin(BaseModel):
+        try:
+            model = LlmProviderModel.objects.get(id=info.data.get("llm_provider_model_id"))
+            if params_cls := LLM_MODEL_PARAMETERS.get(model.name):
+                return params_cls.model_validate(
+                    value or {},
+                    context={
+                        "model_max_token_limit": model.max_token_limit,
+                        "temperature": info.data.get("llm_temperature"),
+                    },
+                ).model_dump()
+        except (LlmProviderModel.DoesNotExist, Exception):
+            pass
```

Ruff (0.14.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)